### DISCUSSION TITLE
Documentation: writing extensions

### DISCUSSION THREAD

**Discussion by ghost** [mhammond/pywin32#2516] [General]
I am missing a point in the documentation.

I would like to write an extension function in C for 
Python on Windows. This function will hold a \*IDispatch 
pointer and would like to return it to Python. That is, it 
has to create a Dispatch object with it \(dynamic or 
static\). I do not see how.

Practically, my intent is to embed Python into an 
application. Embedded scripts should be able to access 
the database through ADO. I would like to pass the 
ADO connection object to Python.

Please consider to make this point clearer in the 
documentation and/or to provide an easy method to 
create a wrapper given a \*IDispatch.

Thanks a lot.

Gael Fraiteur

Reported by: gfraiteur

Original Ticket: "pywin32/feature-requests/14":https://sourceforge.net/p/pywin32/feature-requests/14

**Comment by Avasam** [comment]
Closing as original poster does not exist
